src/components/Router/StepComponentMap.js

```diff
@@ -33,12 +33,11 @@ const createDocumentComponents = (documentType) => {
   return frontDocumentFlow
 }
 
-const crossDeviceComponents = {
-  CrossDeviceLink: () => [CrossDeviceLink],
-  mobileConnection: () => [MobileFlow]
-}
+const crossDeviceComponents = (hasComplete) => ({
+  CrossDevice: () => [CrossDeviceLink, MobileFlow, hasComplete ? Complete : null]
```

Comment:
is adding null to this list a good idea? We do some routing based on the length of this list which assumes everything in it is a component

Reply:
Yes, good point. I've refactored it